src/utils/embeds/serverEmbed.ts

```diff
@@ -19,114 +19,90 @@ type Options = {
  * @param options Options of the embed.
  * @returns Embed that contains the guild info.
  */
-export default async function serverEmbed(options: Options) {
-  const db = await database();
-
+export async function serverEmbed(options: Options) {
   const page = options.page;
   const pages = options.pages;
-
-  // Retrieve guild information
   const guild = options.guild;
   const { premiumTier: boostTier, premiumSubscriptionCount: boostCount } = guild;
   const iconURL = guild.iconURL();
 
-  // Getting the invite table
-  const showInvite = options.showInvite;
-  const serverbTable = await getServerboardTable(db);
-  const invite = await serverbTable?.get(`${guild.id}.invite`).then(
-    async (invite) => invite ? String(invite) : null
-  ).catch(() => null);
+  const invite = (await getServerboardTable(await database()))
+    ?.get(`${guild.id}.invite`)
```

Comment:
```suggestion
  const invite = (await getServerboardTable(await database()))?
    .get(`${guild.id}.invite`)
```
That character belongs there I think ;)